.github/workflows/test.yaml

```diff
@@ -10,8 +10,8 @@ on:
     branches:
       - develop
 jobs:
-  make-package:
-    name: Make Package
```

Comment:
After discussion in Slack, I'm trying to fully understand GitHub workflows/Actions setup.
Specifically the `workflow/make-package.yaml` connection to `workflow/scripts/make-package`. If we are re-naming this to `Test`, do we also need to rename the script to `Test`?

Reply:
Yes, the `run` entries refer to scripts in the `scripts/` directory. Maybe they can be other things (maybe arbitrary shell commands?), I'm not sure. The scripts can be shared between different workflows, so provide a good way to keep the overall Actions codebase DRY.